Source/Plugin.BLE/Shared/AdapterBase.cs

```diff
@@ -226,7 +226,7 @@ public async Task ConnectToDeviceAsync(IDevice device, ConnectParameters connect
         /// Disconnects from the <paramref name="device"/>.
         /// </summary>
         /// <param name="device">Device to connect from.</param>
-        public Task DisconnectDeviceAsync(IDevice device)
+        public Task DisconnectDeviceAsync(IDevice device, CancellationToken cancellationToken = default)
```

Comment:
The GHA build shows a warning here:
```
Parameter 'cancellationToken' has no matching param tag in the XML comment for 'AdapterBase.DisconnectDeviceAsync(IDevice, CancellationToken)' (but other parameters do)
```
@srruu Could you please fix this warning by documenting the `cancellationToken` parameter in the comment above the method?